YoCode/FrontEndCheck.cs

```diff
@@ -4,9 +4,11 @@
 using System.Text;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Chrome;
```

Comment:
This isn't used yet in the code, is it? Or is there another change coming soon that will use this?

Reply:
Will add support for chrome browser, so that if firefox is not installed, program will try to use chrome